.golangci.yaml

```diff
@@ -1,4 +1,6 @@
 linters:
+  run:
+    timeout: 5m
```

Comment:
The `run` configuration is a top level field. It should be specified as follow:

```yaml
run:
  timeout: 5m
linters:
...output-omitted...
```

Reference: https://github.com/golangci/golangci-lint/blob/master/.golangci.yml